src/common/AndroidTypes.ts

```diff
@@ -258,6 +274,9 @@ export class AndroidVirtualDevice {
         this.apiLevel = apiLevel;
     }
 
+    /**
+     * A string representation of an AndroidVirtualDevice which includes Display Name, Device Name, API
```

Comment:
No it's actually API and not API level (although they're related). API would be strings like "Android 9.0 Pie" where as API level for that string would be "API 28". But I know that in different parts of AVD Manager instead of API they call it Target so maybe I'll rename this to be Target.